src/AddTodoForm.js

```diff
@@ -1,11 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+const AddTodoForm = (props) => {
+
+  /*  const [inputData, setInputData] = useState('')
```

Comment:
In the future, be careful with leaving commented code in your codebase unless it adds value to understanding the code. The reason that we are careful with this because it can make your code base hard to navigate esp if you have 10,000+ lines of code. If you want to keep a record of what you commented, you can commit your commented code then remove it. This will leave a trail of the code if you want to reference back to it.